geometry_msgs/Point pt
std_msgs/String command
int16 leg_index
---
geometry_msgs/Point result
